app/src/main/res/drawable/ic_right_arrow.xml

```diff
@@ -1,9 +1,9 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-    android:width="10dp"
-    android:height="11dp"
-    android:viewportWidth="10"
-    android:viewportHeight="11">
+    android:width="16dp"
```

Comment:
This is being used home fragment with different color. This change will affect home fragment UI.